CHANGES.rst

```diff
@@ -22,9 +22,9 @@ Bug fixes
 
   *Related issues and pull requests on GitHub:*
   :issue:`8097`, :issue:`8098`.
-  
-  
-  
```

Comment:
@webknjaz Atleast pre-commit fixes it automatically when I forget.

Reply:
Ah, good to know.